verl/utils/import_utils.py

```diff
@@ -101,6 +101,28 @@ def load_extern_type(file_path: Optional[str], type_name: Optional[str]):
     return getattr(module, type_name)
 
 
+def load_type_from_module(module_path: str, type_name: str) -> Type:
```

Comment:
please add an unit test for this function, since it looks useful and can potentially be reused in other places in the codebase

Reply:
maybe we should just extend `load_extern_type` with added module_path support

Reply:
something like this was just merged in https://github.com/volcengine/verl/pull/2326 I have moved to using that.

Reply:
actually load_extern_type just added support for `pkg://verl.utils.dataset.rl_dataset` that uses import_module`. So it should suffice your need?

Reply:
I just noticed that due to a merge conflict